cli/internal/nodeprovider/local/createNodeIfNotExists.go

```diff
@@ -26,19 +26,15 @@ func (np nodeProvider) CreateNodeIfNotExists(
 		return nil, err
 	}
 
-	nodeProcess, err := pidfile.TryGetProcess(
-		ctx,
-		np.config.DataDir,
-	)
-	if err != nil {
-		return nil, err
-	}
-
-	if nodeProcess != nil {
+	// check if node API reachable
+	if err := apiClientNode.Liveness(ctx); err == nil {
```

Comment:
replacing pidfile.TryGetProcess with apiClientNode.Liveness because the former only works as root and otherwise returns nothing whereas the latter works whether root or not.